ReportingServicesTools/Functions/CatalogItems/Rest/Out-RsRestCatalogItem.ps1

```diff
@@ -5,10 +5,10 @@ function Out-RsRestCatalogItem
 {
     <#
         .SYNOPSIS
-            This command downloads catalog items from a report server to disk. It is for SQL Server Reporting Service 2016 and later.
+            This command downloads catalog items from a report server to disk (using REST endpoint). It is for SQL Server Reporting Service 2016 and later.
```

Comment:
As this uses v2 as default I would change it to say 2017 (and add the note for 2016 use v1)

Reply:
Great point. Removed "It is for SQL Server Reporting Services 2016 and later" and added use "V1 for 2016 under the parameter description for RestApiVersion.